t/lib/Test/Resty.pm

```diff
@@ -87,6 +87,18 @@ sub run_test ($) {
         $cmd .= " $luafile"
     }
 
+    my $perlfile;
+    if (defined $block->perlsrc) {
+        my ($out, $perlfile) = tempfile("testXXXXXX",
+                                        SUFFIX => '.pl',
+                                        TMPDIR => 1,
+                                        UNLINK => 1);
+        print $out ($block->perlsrc);
+        close $out;
+        qx(chmod +x $perlfile);
```

Comment:
Please use the Perl builtin function `chmod` here instead of running a shell command :)